lib/helpers/notifications.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:typed_data';
+import 'package:flutter/material.dart';
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
+import '../main.dart';
+
+class LocalNotifications {
+  // Schedules a notification that specifies a different icon, sound and vibration pattern
+  Future<void> scheduleNotification(minute,sessionName) async {
+    var scheduledNotificationDateTime =
+    DateTime.now().add(Duration(minutes: minute));
+
+    var vibrationPattern = Int64List(4);
+    vibrationPattern[0] = 0;
+    vibrationPattern[1] = 1000;
+    vibrationPattern[2] = 5000;
+    vibrationPattern[3] = 2000;
+
+    var androidPlatformChannelSpecifics = AndroidNotificationDetails(
+        'your other channel id',
+        'your other channel name',
+        'your other channel description',
```

Comment:
It looks like a copy from some sample. See if this can be modified according to our app.